lib/documents.js

```diff
@@ -164,6 +168,56 @@ export async function register({
   }
 }
 
+/**
+ * Encrypts the document either once (with the given recipients) or recursively
+ * in a chain (using the recipientChain).
+ *
+ * @param {object} options - Options hashmap.
+ * @param {object} options.document - The document to register.
+ * @param {Array<string>} recipients - See `register()` docstring.
+ * @param {Array<Array<string>>} recipientChain - See `register()` docstring.
+ *
+ * @returns {Promise<object>} Resolves with the JWE, the result of encryption.
+ */
+export async function encrypt({document, recipients, recipientChain}) {
+  if(!(recipients || recipientChain)) {
```

Comment:
We probably want to throw an error if both are given. We should also move this to the main function and run it once so we don't have to do validation in the helper.

Reply:
Thanks, moved.